packages/client/utils/draftjs/convertToTaskContent.ts

```diff
@@ -1,11 +1,17 @@
 import {ContentState, convertToRaw} from 'draft-js'
 import entitizeText from './entitizeText'
+import isAndroid from './isAndroid'
 
-export const removeSpaces = (str: string) =>
-  str
-    .split(/\s/)
-    .filter((s) => s.length)
-    .join(' ')
+export const removeSpaces = (str: string) => {
+  if (isAndroid) {
+    return str.replace(/ +(?= )/g, '').trim()
```

Comment:
Would this function give different result on non-android devices? 

If this function works identically on all devices, then let's leave only this function for all cases and remove `isAndroid` check.

Reply:
@igorlesnenko This function only works for Android devices; otherwise, we thought it would be ideal to have a single function, but we couldn't.

Reply:
@gitstart-parabol what result it will return on non-android devices? How would it be different from android devices?